requirements.txt

```diff
@@ -79,7 +79,7 @@ selenium-requests==2.0.0
 afterglowpy==0.7.3
 gcn-kafka==0.3.0
 confluent_kafka==1.9.2 # dependency of gcn-kafka: pinning to understand memory effects on GCNEvents
-iminuit==2.18.0
+iminuit==1.5.4
```

Comment:
this seems like a large downgrade. Can you explain why this is needed?

Reply:
> Looks good. You've changed some of the front end so probably work looking at updating the analysis tests in `tests/frontend/sources_and_followup_etc/test_sources.py`
> 
> If you want to test your code locally (using `make run_testing`) you can add the service port to `test_config.yaml`.
> 
> Might be useful to add a section about what services are available out of the box with skyportal in `doc/analysis.md`.

Hi @profjsb, thanks a lot for the review. I do agree that some documentation would be very useful, as well as maybe some tests. I'll work on that.

Reply:
pysedm unfortunately doesn't pin requirements at the moment, and upgrading iminuit breaks it. What I can do is play around with different versions between 1.5.4 and 2.18.0 to figure out the latest version that works. I opened an issue on their repo and they will add a requirements.txt file soon.